docs/2023/DIKSHA/Digital Learners Passbook/updates/2023-09-01.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Week 9
+author: Abhirup Roy
+---
+
+## Milestones
+- [ ] Give the description about Milestone 1
+- [ ] Give the description about Milestone 2
+- [ ] Give the description about Milestone 3
+- [ ] Give the description about Milestone 4
+
+## Screenshots / Videos 
+
+## Contributions
+
+## Learnings
```

Comment:
@abhirupr123 please fix the name of images to a proper one

Reply:
> @abhirupr123 please fix the name of images to a proper one

I have changed the image and video names